armada-ths/ais
# Api image fix

Fixed (the quick and dirty solution) the images in API.

Someone should probably write the code that can take an image and make a formatted copy.

to test this:
* make sure you have an event that has a picture and is published
* goto: localhost:8000/api/events/ and see that it gives a valid url (not "missing")
* optionally do the same for profiles
